src/components/procedure/_shouldafound.mdx

```diff
@@ -38,6 +40,8 @@ This example guides you through steps to report false negatives with Semgrep CLI
     OK to send? [y/N]: y
     ```
 
+    **Note**: Anonymize the data you send to us, as `shouldafound` creates Semgrep Playground links which are public to anyone with access to the URL.
```

Comment:
Is this necessary? Seems like an optional step.

Reply:
I'd like to see it stay in - we wanna make sure we're clear about what's happening.